### DISCUSSION TITLE
--progress argument for backing up and restores

### DISCUSSION THREAD

**Discussion by ghost** [borgbase/vorta#975] [Ideas]
I am running Mac OS Big Sur 11.3.1. I am also running Vorta 0.7.5 using borg version 1.1.16.  

It would be great if there was a way to implement the detail of the --progress argument that is used in borg. So that way we could know how long the backups and restores would take until it is finished. 

Currently when I start a backup or restore I how no idea how long it takes until it is finished.

**Comment by m3nu** [comment]
Since Vorta is a GUI app, we won't be adding CLI args except for very exceptional reasons.

Not sure if this info is even available from Borg. Is Borg giving you this kind of status now?

**Comment by ghost** [comment]
When using borg from terminal when I do a restore using the --progress argument it shows the "Percent" then it says extracting and it shows the full path of the file being extracted.

**Reply by m3nu** [reply]
In that case we could probably show it. Let's see if someone is interested in suggesting a pull request or how interest for this feature is in general.
